.gitignore

```diff
@@ -3,3 +3,5 @@
 .python-version
 .vscode
 *.code-workspace
+*.sql.gz
```

Comment:
This line `*.sql.gz` should be l=placed in `build/docker/data/assets/sql/.gitignore` file. Therefore, please can we remove it from here?

Reply:
Matt has added it in his branch and I have removed it from this gitignore